src/collections/_documentation/platforms/cocoa/pass-scope/swift.md

```diff
@@ -0,0 +1,10 @@
+```swift
+let exception = NSException(name: NSExceptionName("My Custom exeption"), reason: "User clicked the button", userInfo: nil)
+let scope = Scope()
+scope.setLevel(.fatal)
+// By explicity just passing the scope, only the data in this scope object will be added to the event
+// The global scope (calls to configureScope) will be ignored
+// Only do this if you know what you are doing, you loose a lot of useful info
```

Comment:
Do we want to be more explicit here, this comment is hard to parse. Suggest "Only do this if you have mastered this SDK, otherwise, you risk losing useful info"